### PR TITLE
auth 4.3.2: docs&secpoll

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/upgrading.rst [hunk 1]
```diff
@@ -67,6 +67,15 @@ Removed features
...
 Please run ``pdnsutil check-all-zones`` to check for incomplete SOAs.
 
+4.3.1 to 4.3.2
+--------------
+
+Latency calculation changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It turned out that average latency calculations in earlier versions used integers instead of doubles, which led to the throwing away of any data points between 'the current average' and 1000ms above it, instead of having those data points affecting the average.
+In 4.3.1 and 4.4.0, we `started using doubles for this <https://github.com/PowerDNS/pdns/pull/9786/files>`__, which means the latency calculation is accurate now.
```
Comment (mnordhoff) [inline docs/upgrading.rst:77]:
```suggestion
In 4.3.2 and 4.4.0, we `started using doubles for this <https://github.com/PowerDNS/pdns/pull/9786/files>`__, which means the latency calculation is accurate now.
```
Was that supposed to be 4.3.1 or 4.3.2?

Comment (Habbie) [reply docs/upgrading.rst:77]: 4.3.2, thanks!

**Comment by jsoref** [comment]
<details><summary>❌ auth 4.3.2: docs&secpoll Spell checking # 1429</summary>

https://github.com/PowerDNS/pdns/actions/runs/624544011

<details><summary>Run actions/checkout@v2.0.0</summary>

  with:
    ref: refs/pull/10147/merge
    fetch-depth: 5
    repository: PowerDNS/pdns
    token: ***
    persist-credentials: true
    clean: true
    lfs: false
</details>

```
Syncing repository: PowerDNS/pdns
Working directory is '/home/runner/work/pdns/pdns'
/usr/bin/git version
git version 2.30.1
Deleting the contents of '/home/runner/work/pdns/pdns'
/usr/bin/git init /home/runner/work/pdns/pdns
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in /home/runner/work/pdns/pdns/.git/
/usr/bin/git remote add origin https://github.com/PowerDNS/pdns
/usr/bin/git config --local gc.auto 0
/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=5 origin +refs/pull/10147/merge:refs/remotes/pull/10147/merge
Error: fatal: couldn't find remote ref refs/pull/10147/merge
The process '/usr/bin/git' failed with exit code 128
Waiting 19 seconds before trying again
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=5 origin +refs/pull/10147/merge:refs/remotes/pull/10147/merge
Error: fatal: couldn't find remote ref refs/pull/10147/merge
The process '/usr/bin/git' failed with exit code 128
Waiting 11 seconds before trying again
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=5 origin +refs/pull/10147/merge:refs/remotes/pull/10147/merge
Error: fatal: couldn't find remote ref refs/pull/10147/merge
Error: The process '/usr/bin/git' failed with exit code 128
```
</details>

#### Possible responses to this GitHub error
* https://github.com/actions/checkout/issues/36 is the generic "you raced a merge" -- But, as of now, this PR isn't closed and has never been closed
  * File a [new issue](https://github.com/actions/checkout/issue/new) against [actions/checkout](https://github.com/actions/checkout)
* https://github.community/t/issue-fetching-new-prs-refs/1039/2 -- is an unhelpful thread referencing this error condition
  * Reference the thread
  * Start a new thread in [How to use Git and GitHub](https://github.community/c/github-help/how-to-use-git-and-github/21)
* File a [support ticket](https://support.github.com/contact)
* Reimplement merge (we're already halfway there given we have conditional logic for checkout)
* Pretend it didn't happen and hope it won't happen again

**Comment by Habbie** [comment]
https://builder.powerdns.com/#/builders/99/builds/1711

https://builder.powerdns.com/#/builders/113/builds/910

**Review by pieterlexis — APPROVED** [review]

**Pull request merged**